core/executor/src/adapter/trie.rs

```diff
@@ -101,12 +101,17 @@ mod tests {
 
         mpt.insert(&key_1, &val_1).unwrap();
         mpt.insert(&key_2, &val_2).unwrap();
+        mpt.commit().unwrap();
```

Comment:
the purpose of this test is to verify that even without calling `commit()`, the followint `get` can fetch the changes made by the preceeding `insert`.  So, this line seems to be unnecessary?